translate/system-test/translate.test.js

```diff
@@ -22,45 +22,57 @@ const run = require(`../../utils`).run;
 const cwd = path.join(__dirname, `..`);
 const cmd = `node translate.js`;
 const text = `Hello world!`;
+const text2 = `Goodbye!`;
 const toLang = `ru`;
 
 describe(`translate:translate`, () => {
-  const translate = Translate({
-    key: process.env.TRANSLATE_API_KEY
-  });
-  if (!process.env.TRANSLATE_API_KEY) {
-    process.stdout.write(`Skipping Translate API tests...\n`);
-    return;
-  }
+  const translate = Translate();
 
-  it(`should detect language`, (done) => {
+  it(`should detect language of a single string`, () => {
     const output = run(`${cmd} detect "${text}"`, cwd);
-    translate.detect(text, (err, result) => {
-      assert.ifError(err);
-      assert.equal(output, `Detected: ${JSON.stringify(result)}`);
-      done();
-    });
+    return translate.detect(text)
+      .then((results) => {
+        const expected = `Detections:\n${text} => ${results[0].language}`;
+        assert.equal(output, expected);
+      });
+  });
+
+  it(`should detect language of multiple strings`, () => {
+    const output = run(`${cmd} detect "${text}" "${text2}"`, cwd);
+    return translate.detect([text, text2])
+      .then((results) => {
+        const expected = `Detections:\n${text} => ${results[0][0].language}\n${text2} => ${results[0][1].language}`;
```

Comment:
It looks like we're relying on the Translate API to correctly detect `language` here. Is that OK, or should we specify `language` as a constant? (i.e. `"en"`)